src/connection.js

```diff
@@ -929,6 +997,19 @@ class Connection extends EventEmitter {
     process.nextTick(boundTransitionTo, this.STATE.SENT_NTLM_RESPONSE);
   }
 
+  sendFedAuthResponsePacket() {
+    const accessTokenLen = Buffer.byteLength(this.fedAuthInfo.token.accessToken, 'ucs2');
+    const data = new WritableTrackingBuffer(4 + accessTokenLen);
```

Comment:
What do you think about using a regular `Buffer` instead of a `WritableTrackingBuffer` here? The length of the data is fixed, so `WritableTrackingBuffer` just adds bookkeeping overhead without providing any real advantages.